src/app/publications/page.tsx

```diff
@@ -13,22 +14,50 @@ import Sidebar from '@/components/SideBar'
 import Divider from '@/components/Divider'
 import { ScreenSize, linearlyScaleSize } from '@/app/theme'
 
+const CenteredContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
+
 const Container = styled.div`
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   padding: 40px
     ${linearlyScaleSize({
-      minSizePx: 48,
+      minSizePx: 24,
       maxSizePx: 96,
       minScreenSizePx: parseInt(ScreenSize.md),
       maxScreenSizePx: parseInt(ScreenSize.lg),
     })};
+  padding-right: ${linearlyScaleSize({
+    minSizePx: 24,
+    maxSizePx: 8,
+    minScreenSizePx: parseInt(ScreenSize.sm),
+    maxScreenSizePx: parseInt(ScreenSize.md),
+  })};
+  width: 100%;
   gap: 16px;
+  @media (min-width: ${ScreenSize.max}) {
+    width: ${ScreenSize.max};
+    margin: 0 auto;
+  }
+
+  & > main {
```

Comment:
can we just move `main` outside of `Container` and set the width of `Container` to 85%?
We should avoid `>` selectors as much as possible because it makes components less modular and harder to track what affects what.

Reply:
I don't think it would be good to move `main` outside of `Container` because, semantically, `main` should only include the publications content and not the `Sidebar` component.
We could get rid of the pseudoselector and just move the `width` property in `main`'s `style` prop.